cloud/aws/templates/aws_oidc/aim.tf

```diff
@@ -0,0 +1,34 @@
+resource "aws_iam_role" "apprunner_instance_role" {
```

Comment:
why is this here instead of the aws directory

Reply:
added todo at the top.